resources/views/Emails/master.blade.php

```diff
@@ -2,6 +2,18 @@
 <html lang="en">
     <head>
         <meta charset="utf-8">
+        <!-- add styles -->
+        <style>
+            .table-custom {
+                table-layout : fixed;
+                border-collapse: collapse;
+                border-spacing: 0;
+                width: 100%;
+            }
+            .border-custom {
```

Comment:
These would do better with a more descriptive name than `-custom`. What's custom about them? What do they look like? Or what purpose are they for?